packages/salesforcedx-apex-replay-debugger/test/integration/adapter.test.ts

```diff
@@ -234,6 +233,7 @@ describe('Replay debugger adapter - integration', function() {
         classStaticVarsAValidLines
       );
 
+      // tslint:disable-next-line:no-floating-promises
```

Comment:
Could you provide some context on why you choose to ignore the next line?

Reply:
This is the function in question. TS is flagging this as a floating promise. 

```public configurationDoneRequest(
    response: DebugProtocol.ConfigurationDoneResponse,
    args: DebugProtocol.ConfigurationDoneArguments
  ): void {
    if (this.logContext.getLaunchArgs().stopOnEntry) {
      // Stop in the debug log
      this.logContext.updateFrames();
      this.sendEvent(new StoppedEvent('entry', ApexReplayDebug.THREAD_ID));
    } else {
      // Set breakpoints first, then try to continue to the next breakpoint
      this.continueRequest({} as DebugProtocol.ContinueResponse, {
        threadId: ApexReplayDebug.THREAD_ID
      });
    }
  }
```

Reply:
I've been reading up a bit more on the debug adapter protocol. I think that is what lay at the core of this being seen as a promise by TS. Specifically in the debugadapter-testsupport module: 

> The module provides a toolkit with **_Promise-based_** building blocks for individual protocol requests (e.g. `stepInRequest`) and for common request sequences (e.g. `hitBreakpoint`). These building blocks can be easily configured for a specific adapter and combined to form complex scenarios.

Reply:
If it is a promise then it should be handled as such. However, for this one, I _think_ I do agree that this might be a misclassification and it's better to just tell tslint to ignore the line.